dotnet/src/Connectors/Connectors.Memory.Weaviate/Diagnostics/Verify.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.SemanticKernel.Connectors.Memory.Weaviate.Diagnostics;
```

Comment:
Same, can use `  <Import Project="$(RepoRoot)/dotnet/src/InternalUtilities/InternalUtilities.props" />` I believe.

Reply:
I've kept this the same as the Qdrant connector.